Source/WebCore/crypto/keys/CryptoKeyHMAC.cpp

```diff
@@ -46,8 +46,7 @@ static size_t getKeyLengthFromHash(CryptoAlgorithmIdentifier hash)
     case CryptoAlgorithmIdentifier::SHA_512:
         return 1024;
     default:
-        ASSERT_NOT_REACHED();
-        return 0;
+        return std::nullopt;
```

Comment:
I think this is equivalent—please change all of these to be the same (either `{ }` or `std::nullopt`) across all the files:

```suggestion
        return { };
```